packages/fxa-content-server/app/scripts/templates/partial/brand-messaging.mustache

```diff
@@ -9,28 +9,28 @@
                 </div>
             </div>
             <div class="flex-initial max-w-md pr-2">
```

Comment:
Can we use an SVG in the future for the "M" logo? It looks a tad blurry on my MBP screen.

Reply:
Switched to SVG and the pixel css has been replaced with tailwind classes.